libs/designsystem/src/lib/components/item/item.component.scss

```diff
@@ -88,6 +88,14 @@
   }
 }
 
+.disclosure {
+  text-align: center;
```

Comment:
why are we setting text-align here? To me it seems like it has no effect, it is overwritten by ionic-specific text-align anyway.

<img width="237" alt="image" src="https://user-images.githubusercontent.com/42470636/205290360-7b4c841b-0a0c-402e-9b3b-1f79bbc032e1.png">

Reply:
Removed